### ISSUE TITLE
Add a way to extends Convertor

### ISSUE THREAD

**Issue by TheBlusky** [nhn/tui.editor#615]
With the new `customConvertor` parameter, enabling usage of... a custom convertor, how to easily "extends" default one ?

I think it will be nice to be able to import Convertor, so we could write something like

```javascript
import { Convertor } from 'tui-editor'

class MyConvertor extends Convertor {
    toHTML(string) {
        toHTML = super.toHTML(string)
        /* Do custom stuff */
        return toHTML
    }
}
```
Is it possible, or is there already any way to do something like that ?

## Version
1.4.6

## Test Environment
Does not apply

## Current Behavior
Does not apply

**Comment by seonim-ryu** [comment]
@TheBlusky 
The editor does not export `Convertor`, so it will not be imported as below. 🤔 

```js
import { Convertor } from 'tui-editor'
```
Currently, the convertor input through `customConvertor` option is used as it is. In other words, it doesn't inherit from the default convertor.

I think the custom convertor should be the way to inherit the default convertor. And this should be done in the editor. I'll consider how to deal with it.

**Comment by TheBlusky** [comment]
I know it won't work as is :-)

Retrieving the original Convertor class is important because it's allows you to use `super()`.

I worked arround using the following :

```
const dummyEditorInst = new Editor({ el: document.querySelector('#dummy') })

const Convertor = dummyEditorInst.convertor.constructor

class MyConvertor extends Convertor {
  toHTML (string) {
    let toHTML = super.toHTML(string)
    toHTML = customFunction(toHTML)
    return toHTML
  }
```

(Adding a `<div style="visibility: hidden" id="dummy" />` in my dom).

I know, it's ugly, but it's working while having no way to import the Convertor :-)

**Comment by seonim-ryu** [comment]
@TheBlusky
Right. It's so ugly and doesn't seem to be usable. 😢

I'll have to think about how to provide the instance method(API) to get the original converter, or how to handle the custom converter inside the editor so that it can inherit the original converter.  I'll register this as a feature and work on it.

Thank you for your report.

**Comment by dala00** [comment]
I also need. I want to use default Convertor and convert markdown before converted to HTML.

When I convert html of around code in previewBeforeHook, the code highlight is not enabled there. So I need to convert markdown before converted.

**Comment by nsrosenqvist** [comment]
Any update on this? I need this too. A hook like mentioned by @dala00, both when converting to and from HTML, would be a great alternative for my use case.

**Comment by wilsonssss** [comment]
@seonim-ryu  we need this, thanks

**Comment by trollderius** [comment]
That's all I got. Exposing Convertor will be ideal.
```
const editor = new tuiEditor({
  el: document.createElement("div")
});

editor.convertor.toHTML(str);
```

**Comment by dala00** [comment]
No, we may want to convert when body changed immediately. So handler is needed.
